project.clj

```diff
@@ -119,8 +119,7 @@
                        :logrotate-enabled false}
                 :resources {:dir "tmp/ezbake-resources"}
                 :config-dir "ezbake/config"
-                :system-config-dir "ezbake/system-config"
-                :additional-uberjars [[puppetlabs/jruby-deps ~jruby-version]]}
```

Comment:
We might want to still call it out at the top, but I think w/o this line we, can if we want, inline the jruby-version string now.

Reply:
Might as well inline it. Keep it cleaner.

Reply:
Updated.